packages/tooling/cfp-template/README.md

```diff
@@ -0,0 +1,3 @@
+# Introduction
```

Comment:
curious about the file name.  What is 'cfp'?  It's not a common acronym I know.  Can we instead use a common industry acronym or not abbreviate for better code maintainability.

Reply:
It stands for "Create Fast Project", a naming convention I liked from reacts Create React App "cra-template" approach.

Reply:
I've updated the README to include the acronym.

Reply:
^ This is the case with exceptions where using existing convention would cause friction with tools. I think that is an important caveat here. An example - our eslint package name doesn't coincide with _the actual package name in `package.json`_, because we went with convention rather than understanding that there is a prescribed naming approach to ensure packages can be used as eslint plugins. I'm all for consistency until it makes meeting our goals more difficult.

I think intent is necessary in understanding the "why" here. @janechu can correct me here, but one big reason to not be verbose is that I can install this template from the command line and verbosity is not our friend in that case. Even in the case that we are invoking this under the covers, a code comment accomplishes the goal, a README helps onboard folks to the project on it's purpose, and I believe that README's are also helpful for search results, etc. 

I think that as a rule we should follow conventions for package naming, but there are exceptions - specifically when the name of that package is necessary for meeting specific goals or product outcomes. In that case, like with our eslint package, the convention actually creates more of a mess/disconnect.

Reply:
Agree with Chris and Jane on this. We're following some conventions based on other patterns in CLIs and also don't want this to be too long if it needs to be used on the command-line.